assertj-core/src/main/java/org/assertj/core/api/ThrowableAssert.java

```diff
@@ -68,6 +68,7 @@ public static Throwable catchThrowable(ThrowingCallable shouldRaiseThrowable) {
   }
 
   @SuppressWarnings("unchecked")
+  @Deprecated(since = "3.23.11")
   public static <THROWABLE extends Throwable> THROWABLE catchThrowableOfType(ThrowingCallable shouldRaiseThrowable,
                                                                              Class<THROWABLE> type) {
     Throwable throwable = catchThrowable(shouldRaiseThrowable);
```

Comment:
I suggest replacing the body of this method with:

```java
return ThrowableAssert.catchThrowableOfType(type, shouldRaiseThrowable);
```

Reply:
This has been done, please resolve the conservation so the PR can move forward.

Reply:
I don't see it done but I'll do it now 😉